### PR TITLE
Clean up extra jars in build.image/wlp and other cleanup

### PR DESCRIPTION
**Pull request by jhanders34**
- Change a project to not have a transformed jar that isn't used by any features
- Delete two projects that aren't used for anything.  One was replaced with an io.openliberty version and was not cleaned up.  The other was created, but never used.
- Update a project to be a test.project so that it doesn't publish to build.image/wlp/lib directory
- Update dependabot pom.xml for new dependency that didn't get added.

### CODE REVIEW

**Comment by jhanders34** [comment]
#build

File: dev/com.ibm.ws.net.oauth.jsontoken/bnd.bnd [hunk 1]
```diff
@@ -12,4 +12,6 @@
 #*******************************************************************************
 -include= jar:${fileuri;${repo;com.googlecode.jsontoken:jsontoken;1.1.0.ibm-s20140416-2113;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
--buildpath: com.googlecode.jsontoken:jsontoken;strategy=exact;version=1.1.0.ibm-s20140416-2113
\ No newline at end of file
+-buildpath: com.googlecode.jsontoken:jsontoken;strategy=exact;version=1.1.0.ibm-s20140416-2113
+
+test.project: true
```
Comment (jhanders34) [inline dev/com.ibm.ws.net.oauth.jsontoken/bnd.bnd]:
```suggestion
test.project: true
generate.replacement: true
```

**Comment by jhanders34** [comment]
#libby #build

**Review by ayoho — APPROVED** [review]

**Comment by jhanders34** [comment]
#libby

**Review by Azquelt — COMMENTED** [review]
OpenAPI changes look correct. The transformation is removed for the MP OpenAPI 1.x servlet project which is only used with EE 7 & 8 and so doesn't need a Jakarta version.

Thank you.

**Pull request merged**